example/full/instance.properties

```diff
@@ -1062,44 +1062,68 @@ sleeper.default.parquet.writer.version=v2
 
 # The number of attempts to make when applying a transaction to the state store. This default can be
 # overridden by a table property.
-sleeper.default.table.metadata.add.transaction.max.attempts=10
+sleeper.default.metadata.add.transaction.max.attempts=10
 
```

Comment:
I think this should specify that this property is specific to the transaction log state store - should it be 'sleeper.default.metadata.transactionlog.add.transaction.max.attempts'? And it's a separate issue but should 'metadata' here be 'statestore'?